### PR TITLE
welcome page, login, session and dashbooard

### PR DESCRIPTION
**Pull request by JBelta**
Reviewed travis errors.

### CODE REVIEW

**Review by Jonathan-M-Wilson — APPROVED** [review]

File: spec/factories/users.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    name { "MyString" }
```
Comment (carson-jardine) [inline spec/factories/users.rb:3]: I'm not a pro at Factory Bot, so just genuinely curious whether it is best practice to leave the "MyString" or replace it with something like` {  Faker::Name.first_name }` ?

File: spec/support/module/omniauth_macros.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+module OmniauthMacros
+  def stub_omniauth
```
Comment (carson-jardine) [inline spec/support/module/omniauth_macros.rb:2]: This is awesome!

**Review by carson-jardine — APPROVED** [review]
Had a question about FactoryBot best practices but besides that, this looks great. Nice work team!

File: spec/factories/users.rb (hunk 1 above)
Comment (ETBassist) [reply spec/factories/users.rb:3]: Believe it or not, this is actually autogenerated by Factorybot! We haven't changed it all, but adding Faker in is probably a good idea

File: spec/support/module/omniauth_macros.rb (hunk 2 above)
Comment (ETBassist) [reply spec/support/module/omniauth_macros.rb:2]: Just realized that we never actually used this lol, switched the test over to this method

Comment (carson-jardine) [reply spec/support/module/omniauth_macros.rb:2]: lmao I noticed the method but not that we didn't use it 🤦 💀

**Review by carson-jardine — APPROVED** [review]
Noice

File: spec/factories/users.rb (hunk 1 above)
Comment (jdcarey128) [reply spec/factories/users.rb:3]: I think for now it is ok. If there is a situation where we need to create multiple records of a user, then we can implement Faker for these fields

Comment (s-steel) [reply spec/factories/users.rb:3]: I prefer to use faker with factory bot, like Carson was saying, but we can worry about that later.

**Review by s-steel — APPROVED** [review]

File: app/controllers/dashboard_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class DashboardController < ApplicationController
+
```
Comment (jdcarey128) [inline app/controllers/dashboard_controller.rb:2]: I think we'll want to add some sort of before_action check valid user so not anyone can access dashboard paths

**Review by jdcarey128 — APPROVED** [review]
I will create an issue for the small suggestion regarding validating users for the dashboard controller. We can add this later. Otherwise, looks good!

**Pull request merged**